src/shared/lib/StoreUtils.ts

```diff
@@ -231,7 +231,7 @@ export async function fetchAllReferenceGenomeGenes(
     genomeName: string,
     client: CBioPortalAPI = defaultClient
 ) {
-    if (AppConfig.serverConfig.app_name === 'public-portal') {
+    if (AppConfig.serverConfig.show_reference_genome_genes) {
```

Comment:
this is some temp performance feature, maybe `feature_flag.use_static_reference_genome_genes`?

Reply:
It looks like this line was replaced in a new commit since I last updated it. I will just remove my flag altogether.